examples/CAgent/c_agent/agent.py

```diff
@@ -69,7 +69,8 @@
 __version__ = 1.0
 
 '''This example agent calls functions from a shared object via
```

Comment:
docstrings should be """
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings